saleor/graphql/plugins/sorters.py

```diff
@@ -1,30 +1,43 @@
-from typing import TYPE_CHECKING, List, Optional
+from typing import List, Optional
 
 import graphene
 
+from ..core.enums import OrderDirection
 from ..core.types import SortInputObjectType
+from .types import Plugin
 
-if TYPE_CHECKING:
-    # flake8: noqa
-    from ...plugins.base_plugin import BasePlugin
+
+def sort_active_key(plugin: Plugin, sort_reverse: bool):
```

Comment:
The same question as in filter? 
For example in a product to sort through any channel filed we should provide a channel for which we want to sort data.

Reply:
We are currently discussing how this should be handled, but it will be added in separate PR.